CommonTools/RecoAlgos/python/vertexCompositeCandidateCollectionSelector_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+from CommonTools.RecoAlgos.VertexCompositeCandidateCollectionSelector_cfi import VertexCompositeCandidateCollectionSelector as vertexCompositeCandidateCollectionSelector
+
+
+vertexCompositeCandidateCollectionSelector.v0            = cms.InputTag('generalV0Candidates:Kshort') # generalV0Candidates:Lambda
```

Comment:
shouldn't you clone VertexCompositeCandidateCollectionSelector instead of modifying it?
What if someone needs to load VertexCompositeCandidateCollectionSelector_cfi in a different place